### PR TITLE
Feature Other App

### PR DESCRIPTION
**Pull request by NguyenNgocTuyenUED**
Play Video when users switch other applications or not interact this application
https://edu-redmine.sun-asterisk.vn/issues/8852

### CODE REVIEW

**Review by thaohq-0814 — APPROVED** [review]

**Comment by NguyenNgocTuyenUED** [comment]
ready

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -22,4 +29,5 @@
             </intent-filter>
         </activity>
     </application>
+
```
Comment (tuanlvt-1117) [inline app/src/main/AndroidManifest.xml]: remove blank

File: app/src/main/java/com/sun_asterisk/youtubebackground/screen/play/PlayFragment.java [hunk 2]
```diff
@@ -63,43 +64,38 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
...
-                    public void onInitializationSuccess(YouTubePlayer.Provider arg0,
-                            YouTubePlayer youTubePlayer, boolean isFullScreen) {
-                        if (!isFullScreen) {
-                            mYouTubePlayer = youTubePlayer;
-                            mYouTubePlayer.setFullscreen(false);
-                            mYouTubePlayer.loadPlaylist(Constant.PLAYLIST, mPosition, 0);
-                            mYouTubePlayer.play();
-                        }
-                    }
+    private void setUI() {
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/youtubebackground/screen/play/PlayFragment.java]:
```suggestion
    private void setViewYoutubePlayer() {
```

**Comment by NguyenNgocTuyenUED** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**